vars/cleanup.groovy

```diff
@@ -0,0 +1,23 @@
+def call(Map config=[:]) {
+  node {
+    stage('delete resources of closed PR') {
+      cleanup(config.environment)
+    }
+  }
+}
+
+def cleanup(environment) {
```

Comment:
Is it possible to have `repoName` passed into the method rather than it being assumed to be available within the execution scope?